### PR TITLE
Add Gazebo 9.11.0

### PR DESCRIPTION
**Pull request by seanyen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
cc @wolfv @traversaro @j-rivero currently I only tackled the Windows build.

File: recipes/gazebo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  skip: true  # [not win or vc<14]
+  run_exports:
+    - {{ pin_subpackage('gazebo', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
+    - vs2017_win-32                      # [win32]
```
Comment (marcelotrevisani) [inline recipes/gazebo/meta.yaml]: the ``vs`` here is not necessary, as you are already skipping ``vc<14`` just the ``vs2017`` will be used

File: recipes/gazebo/build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```
Comment (marcelotrevisani) [inline recipes/gazebo/build.sh]: the shebang is not necessary, ``conda-build`` already handles it :)
```suggestion
```

**Comment by seanyen** [comment]
cc @conda-forge/staged-recipes this is ready for review and merge.

File: recipes/gazebo/bld.bat [hunk 3]
```diff
@@ -0,0 +1,26 @@
+set PKG_CONFIG_PATH=%LIBRARY_PREFIX%\share\pkgconfig;%LIBRARY_PREFIX%\lib\pkgconfig;%PKG_CONFIG_PATH%
+
+mkdir build
+cd build
+cmake ^
+    -G "NMake Makefiles" ^
```
Comment (xhochy) [inline recipes/gazebo/bld.bat]: Also use `Ninja`  here to utilise parallelism in the CI
```suggestion
    -G "Ninja" ^
```

File: recipes/gazebo/bld.bat [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True ^
+    -DBOOST_ROOT=%LIBRARY_PREFIX% ^
+    -DBoost_NO_SYSTEM_PATHS=ON ^
+    -DBoost_NO_BOOST_CMAKE=ON ^
```
Comment (xhochy) [inline recipes/gazebo/bld.bat]: Except this one, no other Boost option should be necessary

File: recipes/gazebo/build.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+cmake .. \
+      -G "Ninja" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
+      -DBOOST_ROOT=$PREFIX \
+      -DBoost_NO_SYSTEM_PATHS=ON \
+      -DBoost_NO_BOOST_CMAKE=ON \
```
Comment (xhochy) [inline recipes/gazebo/build.sh]: As above, this should suffice

File: recipes/gazebo/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    - ninja                              # [not win]
```
Comment (xhochy) [inline recipes/gazebo/meta.yaml]:
```suggestion
    - ninja
```

**Comment by seanyen** [comment]
@xhochy thanks for the feedback. I updated the pull request and let's see if the build passes.

**Comment by wolfv** [comment]
congrats on getting this in! I'll try to make it work for Linux this week :)

**Pull request merged**